React-Boostrap/ building frontend

Peneloppe-jack opened the pull request:
integrating React Bootstrap into myFlix app and adding consistent styling across each of your views :+1:
issues with PARCEL
depsite switching CORS status

Peneloppe-jack committed:
React-Boostrap/ building frontend

integrating React Bootstrap into myFlix app and adding consistent styling across each of your views :+1:
issues with PARCEL
depsite switching CORS status

Peneloppe-jack committed:
updates

Peneloppe-jack committed:
update

HannaHoJ commented on src/components/login-view/login-view.jsx:
```diff
@@ -0,0 +1,78 @@
+<<<<<<< Updated upstream
+=======
```
i guess u have a merge conflict here. 
this should be probably removed.

```
<<<<<<< Updated upstream
=======
```

HannaHoJ replied:
hmm u have 2 versions currently in ur file, the updated stream version and the stashed changes. which u wanna keep? u can just delete the ones u dont want to keep.

Peneloppe-jack committed:
fixed merge conflicts

johannaasuper star!!

Peneloppe-jack committed:
Merge branch 'main' into myFlix-client-Fronted-React/Bootstrap

# Conflicts:
#	package-lock.json
#	package.json
#	src/components/login-view/login-view.jsx
#	src/components/main-view/main-view.jsx
#	src/components/movie-card/movie-card.jsx
#	src/components/movie-view/movie-view.jsx
#	src/components/registration-view/registration-view.jsx
#	src/index.html
#	src/index.jsx

Peneloppe-jack committed:
mr left over

Peneloppe-jack committed:
fix parcel

Peneloppe-jack committed:
...

...

Peneloppe-jack committed:
fixing the little details

...

Peneloppe-jack committed:
...

ok

Peneloppe-jack committed:
Update registration-view.jsx

Peneloppe-jack committed:
Update login-view.jsx

Peneloppe-jack committed:
Updates

So Starting over :
- all I need at this point correct components and their SCSS pages :
- src / ccomponents /
 - director-view jsx+scss
-genre-view jsx + scss
-login-view jsx+scss
-main-view jsx+scss
-movie-card jsx +scss
-movie-view jsx +scss
-registration-view jsx+scss

index /html/jsx/ scss
package.json up to date !!

Peneloppe-jack committed:
...

..
